lib/maxitest/let_all.rb

```diff
@@ -1,12 +1,18 @@
-Minitest::Spec::DSL.class_eval do
-  def let_all(name, &block)
-    cache = []
-    define_method(name) do
-      if cache.empty?
-        cache[0] = instance_eval(&block)
-      else
-        cache[0]
+module Maxitest
+  module LetAll
+    def let_all(name, &block)
+      cache = []
+      define_method(name) do
+        if cache.empty?
+          cache << instance_eval(&block)
+        end
+        cache.first
       end
     end
+    def self.included(base)
+      base.extend(self)
+    end
```

Comment:
I need to module to show up in MiniTest::Spec::DSL.included_modules.

I will use extend if I was assembling a bigger class/module .  But since we are patching, I think that a visible trace is better . wdyt ?

Reply:
don't think many are using that or caring for it 🤷 
I usually do `.method(:foo).source_location`

Reply:
... also extended_modules kinda exists
```
class A; extend B;end
class << A; included_modules;end
=> [B, Kernel]
```

Reply:
no hard feelings either way, just never use it ...

Reply:
Nice. I will use that then.